app/email_utils.py

```diff
@@ -1457,3 +1462,44 @@ def get_dmarc_status(msg: Message) -> Optional[DmarcCheckResult]:
             return dmarc_result
 
     return None
+
+
+def generate_verp_email(verp_type: VerpType, tx_id: int) -> str:
+    data = [verp_type.bounce.value, tx_id, int(time.time())]
+    json_payload = json.dumps(data).encode("utf-8")
+    # Signing without itsdangereous because it uses base64 that includes +/= symbols and lower and upper case letters.
```

Comment:
We're using hmac to generate the signature. It should be safe. We're doing the same thing JWTs are doing but just restricting the hash to sha224 to reduce the length of the signature and skipping the header because the hash function and method are pre-defined.